qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -127,190 +126,219 @@ def read_structure(filepath, outpath="."):
+            Possible values for ``mult`` are :math:`1, 2, 3, \ldots`. If not specified,
+            a closed-shell HF state is assumed.
+        basis (str): Atomic basis set used to represent the HF orbitals. Basis set
+            availability per element can be found
+            `here <www.psicode.org/psi4manual/master/basissets_byelement.html#apdx-basiselement>`_
+        package (str): Quantum chemistry package used to solve the Hartree-Fock equations.
+            Either ``'pyscf'`` or ``'psi4'`` can be used.
+        outpath (str): Path to output directory
 
     Returns:
-        str: path to the directory containing the file with the Hartree-Fock electronic structure
+        str: full path to the file containing the mean field electronic structure
+
+    **Example**
+
+    >>> name = 'h2'
+    >>> geometry = [['H', (0.0, 0.0, -0.35)], ['H', (0.0, 0.0, 0.35)]]
+    >>> meanfield(name, geometry)
+    ./pyscf/sto-3g/h2
     """
 
-    qc_package = qc_package.strip().lower()
+    package = package.strip().lower()
 
-    if qc_package not in ("psi4", "pyscf"):
-        qc_package_error_message = (
+    if package not in ("psi4", "pyscf"):
+        error_message = (
             "Integration with quantum chemistry package '{}' is not available. \n Please set"
-            " 'qc_package' to 'pyscf' or 'psi4'.".format(qc_package)
+            " 'package' to 'pyscf' or 'psi4'.".format(package)
         )
-        raise TypeError(qc_package_error_message)
-
-    qcp_dir = os.path.join(outpath.strip(), qc_package)
-    path_to_hf_data = os.path.join(qcp_dir, basis.strip())
-
-    if not os.path.isdir(qcp_dir):
-        os.mkdir(qcp_dir)
-        os.mkdir(path_to_hf_data)
-    elif not os.path.isdir(path_to_hf_data):
-        os.mkdir(path_to_hf_data)
-
-    molecule = MolecularData(
-        geometry,
-        basis,
-        multiplicity,
-        charge,
-        filename=os.path.join(path_to_hf_data, mol_name.strip()),
-    )
+        raise TypeError(error_message)
+
+    package_dir = os.path.join(outpath.strip(), package)
+    basis_dir = os.path.join(package_dir, basis.strip())
+
+    if not os.path.isdir(package_dir):
+        os.mkdir(package_dir)
+        os.mkdir(basis_dir)
+    elif not os.path.isdir(basis_dir):
+        os.mkdir(basis_dir)
+
+    path_to_file = os.path.join(basis_dir, name.strip())
 
-    if qc_package == "psi4":
+    molecule = MolecularData(geometry, basis, mult, charge, filename=path_to_file)
+
+    if package == "psi4":
         run_psi4(molecule, run_scf=1, verbose=0, tolerate_error=1)
 
-    if qc_package == "pyscf":
+    if package == "pyscf":
         run_pyscf(molecule, run_scf=1, verbose=0)
 
-    return path_to_hf_data
+    return path_to_file
 
 
-def active_space(mol_name, hf_data, n_active_electrons=None, n_active_orbitals=None):
-    r"""Builds the active space by partitioning the set of Hartree-Fock molecular orbitals.
+def active_space(n_electrons, n_orbitals, mult=1, nact_els=None, nact_orbs=None):
```

Comment:
I have to admit, I find something like `active_electrons` and `active_orbs` a lot more clearer than `nact_els` and `nact_orbs`.

Reply:
Agree, please see my comments above regarding the prefix "n"